index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="styles.css" />
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>TicTacToe</title>
+  </head>
+  <div>
+    <button class="gameSpace"></button>
+    <button class="gameSpace"></button>
+    <button class="gameSpace"></button>
+  </div>
+  <div>
+    <button class="gameSpace"></button>
+    <button class="gameSpace"></button>
+    <button class="gameSpace"></button>
+  </div>
+  <div>
+    <button class="gameSpace"></button>
+    <button class="gameSpace"></button>
+    <button class="gameSpace"></button>
+  </div>
+  <body></body>
```

Comment:
the divs with your game space should probably go in the body of your html file

Reply:
yes this is correct, everything that is not the HEAD should go into the body tags

Reply:
even the footer tag should be in the body